### PR TITLE
Expose /oltp-http/ endpoint to frontendproxy

### LINKED ISSUE #915: Envoy needs to expose a route for the collector to route frontend spans
To aid in getting browser spans into the collector, Envoy should expose the collector HTTP ingest route at /otlp-http binding to otel-collector:4318 and the browser should forward OTLP traffic to it.

Frontend will need to send traffic to this endpoint.

### PR DESCRIPTION
**Pull request by chigia001**
# Changes

Expose /oltp-http/ endpoint to frontend-proxy and config browser to use this endpoint instead of calling otel-collecter container directly

This fix #915

Additional change:

- Introduce OTEL_COLLECTOR_PORT_HTTP to expose HTTP port of OLTP receiver in otel-collector
- rename OTEL_COLLECTOR_PORT to OTEL_COLLECTOR_PORT_GRPC to differentiate with HTTP port

## Merge Requirements

For new features contributions please make sure you have completed the following
essential items:

* [x] `CHANGELOG.md` updated to document new feature additions
* [x] Appropriate documentation updates in the [docs][]
* [x] Appropriate Helm chart updates in the [helm-charts][]

Maintainers will not merge until the above have been completed. If you're unsure
which docs need to be changed ping the
[@open-telemetry/demo-approvers](https://github.com/orgs/open-telemetry/teams/demo-approvers).

[docs]: https://opentelemetry.io/docs/demo/
[helm-charts]: https://github.com/open-telemetry/opentelemetry-helm-charts

### CODE REVIEW

File: src/frontendproxy/envoy.tmpl.yaml [hunk 1]
```diff
@@ -36,6 +36,8 @@ static_resources:
                           route: { cluster: featureflag, prefix_rewrite: "/" }
                         - match: { prefix: "/loadgen" }
                           route: { cluster: loadgen, prefix_rewrite: "/" }
+                        - match: { prefix: "/oltp-http/" }
```
Comment (chigia001) [inline src/frontendproxy/envoy.tmpl.yaml:39]: I need a trailing splash here.
Without it, envoy returns 301 and redirects to the path without `/oltp-http`

File: src/frontendproxy/envoy.tmpl.yaml [hunk 2]
```diff
@@ -57,14 +59,26 @@ static_resources:
...
+        cluster_name: opentelemetry_collector_grpc
         endpoints:
           - lb_endpoints:
               - endpoint:
                   address:
                     socket_address:
                       address: ${OTEL_COLLECTOR_HOST}
-                      port_value: ${OTEL_COLLECTOR_PORT}
+                      port_value: ${OTEL_COLLECTOR_PORT_GRPC}
+    - name: opentelemetry_collector_http
```
Comment (chigia001) [inline src/frontendproxy/envoy.tmpl.yaml:70]: I'm very new to Envoy, so I'm unsure if we need a separate cluster for gRPC and HTTP

**Comment by chigia001** [comment]
@austinlparker not sure if there is any issue with my PR?
Would love to have feedback on this.

**Comment by austinlparker** [comment]
We'll look at this in our SIG meeting today, thank you for the PR!

**Review by puckpuck — APPROVED** [review]
This is awesome. I tested the branch, and it works as expected, with Chrome dev tools showing the new URL for traces from the web UI.

File: src/frontendproxy/envoy.tmpl.yaml (hunk 2 above)
Comment (puckpuck) [reply src/frontendproxy/envoy.tmpl.yaml:70]: I'm not an Envoy expert either, but my understanding is you need multiple clusters to make this happen.

**Pull request merged**